index.html

```diff
@@ -0,0 +1,46 @@
+---
+title: "R9y"
```

Comment:
personal preference:  i feel capital R makes it odd-looking, despite proper-english.  r9y instead?

Reply:
Yeah, no worries, more than happy to make changes to text etc, this is just an idea of what it could look like - those screenshots need a lot of work too! 😂

Reply:
launch and iterate!  happy to merge whenever you're ready.